mantle/kola/tests/misc/network.go

```diff
@@ -59,55 +59,68 @@ type listener struct {
 func checkListeners(c cluster.TestCluster, expectedListeners []listener) error {
 	m := c.Machines()[0]
 
-	output := c.MustSSH(m, "sudo netstat -plutn")
+	output := c.MustSSH(m, "sudo ss -plutn")
 
 	processes := strings.Split(string(output), "\n")
 	// verify header is as expected
-	if len(processes) < 2 {
-		c.Fatalf("expected at least two lines of nestat output: %q", output)
+	if len(processes) < 1 {
+		c.Fatalf("expected at least one line of ss output: %q", output)
 	}
-	if processes[0] != "Active Internet connections (only servers)" {
-		c.Fatalf("netstat output has changed format: %q", output)
-	}
-	if !regexp.MustCompile(`Proto\s+Recv-Q\s+Send-Q\s+Local Address\s+Foreign Address\s+State\s+PID/Program name`).MatchString(processes[1]) {
-		c.Fatalf("netstat output has changed format: %q", output)
+	headerRegex := `Netid\s+State\s+Recv-Q\s+Send-Q\s+Local Address:Port\s+Peer Address:Port\s+Process`
+	if !regexp.MustCompile(headerRegex).MatchString(processes[0]) {
+		c.Fatalf("ss output has changed format: %q", processes[0])
 	}
 	// skip header
-	processes = processes[2:]
+	processes = processes[1:]
+
+	// create expectedListeners map
+	expectedListenersMap := map[listener]bool{}
+	for _, expected := range expectedListeners {
+		expectedListenersMap[expected] = true
+	}
 
 NextProcess:
 	for _, line := range processes {
+		/*
+			Sample `sudo ss -plutn` output:
+			Netid  State   Recv-Q  Send-Q  Local Address:Port  Peer Address:Port  Process \n
+			udp    UNCONN  0       0           127.0.0.1:323        0.0.0.0:*      users:((\"chronyd\",pid=856,fd=5)) \n
+			udp    UNCONN  0       0               [::1]:323           [::]:*      users:((\"chronyd\",pid=856,fd=6)) \n
+			tcp    LISTEN  0       128           0.0.0.0:22         0.0.0.0:*      users:((\"sshd\",pid=1156,fd=5)) \n
+			tcp    LISTEN  0       128              [::]:22            [::]:*      users:((\"sshd\",pid=1156,fd=7))
+		*/
```

Comment:
There are a few `\n` `\"` characters in there. Let's remove those. Also might be a good idea to run this file through `gofmt -w`.

```suggestion
        /*
            Sample `sudo ss -plutn` output:
            Netid  State   Recv-Q  Send-Q  Local Address:Port  Peer Address:Port   Process
            udp    UNCONN  0       0           127.0.0.1:323        0.0.0.0:*      users:(("chronyd",pid=856,fd=5))
            udp    UNCONN  0       0               [::1]:323           [::]:*      users:(("chronyd",pid=856,fd=6))
            tcp    LISTEN  0       128           0.0.0.0:22         0.0.0.0:*      users:(("sshd",pid=1156,fd=5))
            tcp    LISTEN  0       128              [::]:22            [::]:*      users:(("sshd",pid=1156,fd=7))
        */
```